### PR TITLE
revise YahooDmp integration

### PR DESCRIPTION
**Pull request by waysaku**
Finally we've started CookieSync on Yahoo! website. In addition to that TreasureData has been able to integrate Yahoo! DMP API directly from now on.
If you'd like to integrate with Yahoo! DMP, please contact TreasureData support team.

Former integration way is deprecated from now on.

### CODE REVIEW

File: integration-box/yahoo-dmp/scripts/get_presigned_url.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+import requests
+import json
+from logging import DEBUG, INFO, StreamHandler, getLogger
+import digdag
+
+os.system(f"{sys.executable} -m pip install -U pytd==1.0.0 td-client")
```
Comment (chezou) [inline integration-box/yahoo-dmp/scripts/get_presigned_url.py:8]: The latest version of pytd is 1.3.0.

File: integration-box/yahoo-dmp/scripts/get_presigned_url.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "uidKey": uid_key,
+    "brandGuid": brand_guid,
+    "tagFields": {
+      "p": tag_fields_p,
+      "lid": tag_fields_lid
+    }
+  }
+
+  headers = {"x-api-key": os.getenv('x_api_key')}
+  response = requests.post(yahoo_api_url, data=json.dumps(post_data), headers=headers)
```
Comment (chezou) [inline integration-box/yahoo-dmp/scripts/get_presigned_url.py:41]: you can use `json=post_data` instead.

https://requests.readthedocs.io/en/master/user/quickstart/#more-complicated-post-requests

File: integration-box/yahoo-dmp/scripts/put_userlist.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    querytxt = f.read()
+
+  client = pytd.Client(apikey=os.getenv('td_apikey'), database=database)
+  res = client.query(querytxt)
+  df = pd.DataFrame(**res)
+  csv = df.to_csv(index=False)
+
+  requests.put(presigned_url, data=csv)
+
+
```
Comment (chezou) [inline integration-box/yahoo-dmp/scripts/put_userlist.py:20]: [nits] extra line breaks

File: integration-box/yahoo-dmp/README.md [hunk 4]
```diff
@@ -1,26 +0,0 @@
-# Workflow definition to integrate Yahoo DMP
```
Comment (chezou) [inline integration-box/yahoo-dmp/README.md:1]: seems readme is removed while code is renewed. don't we need to have new readme?

**Comment by waysaku** [comment]
@chezou Sorry, I missed your comment. I'll check your worth pointing out.

**Pull request merged**